src/main/resources/modules/renown_cardiology_study_1.json

```diff
@@ -0,0 +1,881 @@
+{
+  "name": "renown_cardiology_study_1",
```

Comment:
new file which will generate the patient population with inclusion and exclusion criteria for the cardiology study #1